packages/react-data-grid/src/__tests__/CellAction.spec.js

```diff
@@ -1,104 +1,103 @@
-      const renderedActionButtonProps = renderedActionButton.props();
-      const renderedActionMenu = wrapper.find('.rdg-cell-action-menu');
-
-      expect(renderedActionButton.length).toBe(1);
-      expect(renderedActionButtonProps.onClick).toEqual(jasmine.any(Function));
-
-      renderedActionButton.simulate('click');
-
-      expect(props.action.callback.called).toBeTruthy();
-
-      expect(renderedActionMenu.length).toBe(0);
-    });
-  });
-
-  describe('when a menu action is passed', () => {
-    it('will render a toggle button which will hide/show the menu of actions', () => {
-      const { wrapper, props } = setup({
-        action: {
-          icon: 'glyphicon glyphicon-link',
-          actions: [
-            {
-              text: 'Test Action 1',
-              callback: sinon.spy()
-            },
-            {
-              text: 'Test Action 1',
-              callback: sinon.spy()
-            }
-          ]
-        }
-      });
-      let wrapperState = wrapper.state();
-      const renderedActionButton = wrapper.find('.rdg-cell-action-button');
-      const renderedActionButtonProps = renderedActionButton.props();
-      let renderedActionMenu = wrapper.find('.rdg-cell-action-menu');
-
-      expect(renderedActionButton.length).toBe(1);
-      expect(renderedActionButtonProps.onClick).toEqual(jasmine.any(Function));
-      expect(wrapperState.isMenuOpen).toBeFalsy();
-      expect(renderedActionMenu.length).toBe(0);
-
-      renderedActionButton.simulate('click');
-
-      wrapperState = wrapper.state();
-      renderedActionMenu = wrapper.find('.rdg-cell-action-menu');
-      const renderedActionMenuProps = renderedActionMenu.props();
-
-      expect(wrapperState.isMenuOpen).toBeTruthy();
-      expect(renderedActionMenu.length).toBe(1);
-      expect(renderedActionMenuProps.children.length).toBe(props.action.actions.length);
-
-      expect(props.action.actions[0].callback.called).toBeFalsy();
-      renderedActionMenu.childAt(0).simulate('click');
-      expect(props.action.actions[0].callback.called).toBeTruthy();
-    });
-  });
-
-  describe('when isFirst is passed', () => {
-    describe('when isFirst is true', () => {
-      it('will render a button action with cell-action-last class on it', () => {
-        const { wrapper } = setup({ isFirst: true });
-        const wrapperProps = wrapper.props();
-
-        expect(wrapper.length).toBe(1);
-        expect(wrapperProps.className).toBe('rdg-cell-action rdg-cell-action-last');
-      });
-    });
-
-    describe('when isFirst is false', () => {
-      it('will render a button action without cell-action-last class on it', () => {
-        const { wrapper } = setup({ isFirst: false });
-        const wrapperProps = wrapper.props();
-
-        expect(wrapper.length).toBe(1);
-        expect(wrapperProps.className).toBe('rdg-cell-action');
-      });
-    });
-  });
-});
+import React from 'react';
```

Comment:
Let's add a gitattributes file in another PR.
https://help.github.com/en/articles/dealing-with-line-endings